examples/all_the_things.py

```diff
@@ -119,5 +119,6 @@ def teardown(test):
   #  with open('my_private_key.json', 'r') as json_file:
   #    test.AddOutputCallbacks(output.UploadToMfgInspector.from_json(
   #        json.load(json_file)))
-  test.Configure(teardown_function=teardown)
-  test.Execute(test_start=triggers.PromptForTestStart())
+  # test.Configure(teardown_function=teardown)
```

Comment:
Aaarrrgg. I make this mistake every single time I'm working on the frontend. I just usually catch it before I make a PR.